### ISSUE TITLE
Image Merge isn't reset properly on current Master

### ISSUE THREAD

**Issue by K-Leon** [SimpleSoftwareIO/simple-qrcode#93]
First of a real great Project!!! Thank you so much for your hard work.

I just found a small glitch. This one results in an image in both barcodes even though it's only expected to be in the first one:
```
<img src="data:image/png;base64,{!! base64_encode(QrCode::format('png')->merge('qrlogo)->errorCorrection('H')->size(70)->margin(0)->generate($content1)) !!}">
<img src="data:image/png;base64,{!! base64_encode(QrCode::format('png')->size(70)->margin(0)->generate($ontent2)) !!}">
```

I'm not really sure how to fix it appropriate. I hacked a reset for now in the generate function, but this isn't really a good place ;)

**Comment by SimplyCorey** [comment]
I think I know what the problem is.  I'll have a patch out this Friday.  Thanks for the report

**Comment by SimplyCorey** [comment]
Fixed in 1.6.0
